server/manager/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java

```diff
@@ -78,7 +80,13 @@ public long isReady(long tid, Master environment) throws Exception {
   }
 
   @Override
-  public void undo(long tid, Master env) {
+  public void undo(long tid, Master env) throws IOException {
+    // Clean up split files if create table operation fails
+    if(tableInfo.getInitialSplitSize() > 0) {
+      Path p = tableInfo.getSplitPath().getParent();
+      FileSystem fs = p.getFileSystem(env.getContext().getHadoopConf());
+      fs.delete(p, true);
```

Comment:
@Manno15 if this throws an Exception, then unreserveNamespace() will not execute and that may not be good.

Reply:
We could wrap this and put the unreserve call in the finally block.

Reply:
I will make a new pull request with these changes

Reply:
Follow up made on #1759